vllm/transformers_utils/configs/dbrx.py

```diff
@@ -151,10 +152,12 @@ def from_pretrained(
             and hasattr(cls, "model_type")
             and config_dict["model_type"] != cls.model_type
         ):
+            logger_data = {"model_type": config_dict['model_type']}
```

Comment:
```suggestion
            logger_data = {
                "event": "pretrained-model-type-mismatch",
                "pretrained_model_type": config_dict['model_type'],
                "target_model_type": cls.model_type,
            }
```

Reply:
Also, I suggest paying attention to code you are working on and using the kind of quote used by that code (`"`) instead of using your preferred quote type (`'`).

In the `vllm` folder, the pattern `/['/` occurs 33 times and the pattern `/["/` occurs 175 times.

This is a good practice to follow as a contributor and can help ensure that your PR doesn't get hung up on trivial inconsistencies.